clients/stellar-relay-lib/src/config.rs

```diff
@@ -8,9 +8,9 @@ use substrate_stellar_sdk::SecretKey;
 /// It configures both the ConnectionInfo and the NodeInfo.
 #[derive(Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct StellarOverlayConfig {
-	stellar_history_base_url: String,
-	connection_info: ConnectionInfoCfg,
-	node_info: NodeInfoCfg,
+	pub stellar_history_archive_urls: Vec<String>,
```

Comment:
possibly `pub(crate)` ? I assumed this was made pubilc for testing reasons.
Not sure if we want the fields exposed outside of `stellar-relay-lib`

Reply:
We actually can't use `pub(crate)` here because this config lives in the `stellar-relay-lib` and the tests I added live in the `vault/oracle` crate 😅

Reply:
oh, my bad. alrighty then, 👍